Sources/MongoSwift/MongoClient.swift

```diff
@@ -187,9 +187,14 @@ public class MongoClient {
             throw UserError.invalidArgumentError(message: "libmongoc not built with TLS support")
         }
 
+        self.encoder = BSONEncoder(options: options)
```

Comment:
was there a reason for moving this and the next line up here?

Reply:
Swift wanted all the other fields to be initialized before using self._client inside the closure.
` error: 'self' captured by a closure before all members were initialized`